src/mdtemplate.py

```diff
@@ -0,0 +1,38 @@
+"""
+Contains the MDTemplate class. This class is used to
+create a Markdown template used to generate a Markdown table.
+"""
+
+
+class Template:
+    """
+    This class is used to format the data into a table in markdown format.
```

Comment:
can we add attributes description if possible?

Reply:
Sure thing, but I think it would be better if we use [`mdutils`](https://mdutils.readthedocs.io/en/latest/) for generating the markdown file, and delete this class. 😅